modules/ob_common/manifests/init.pp

```diff
@@ -87,6 +87,16 @@
     require => File["${product_dir}", "${pack_dir}"],
   }
 
+  # Copy accellerator binary to distribution path
+  file { "wso2-accelerator-binary":
+    path    => "${pack_dir}/${accellerator_binary}",
```

Comment:
Since this is a public repository, shall we correct the spelling of this term in all the occurrences?

Reply:
Corrected, Thanks.